angular/projects/spark-angular/src/lib/directives/sprk-text/sprk-text.stories.ts

```diff
@@ -0,0 +1,125 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkTextModule } from './sprk-text.module';
+import { SprkTextDirective } from './sprk-text.directive';
+import { markdownDocumentationLinkBuilder } from '../../../../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: 'Foundations/Typography',
+  component: SprkTextDirective,
+  decorators: [
+    storyWrapper(
+      storyContent => (
+        `<div class="sprk-o-Box">${storyContent}<div>`
+      )
+    )
+  ],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('typography', 'foundations')}
+
+##### Line-height Collision
+
+By default, line height on type will create extra
```

Comment:
line-height

Reply:
@RCopeland , we have this on the gatsby site like I have it in this PR. https://sparkdesignsystem.com/using-spark/foundations/typography/#gatsby-focus-wrapper - I'm going to keep it like we have it in the gatsby site.